web/pandas/interactive_terminal/README.md

```diff
@@ -0,0 +1,17 @@
+# The interactive `pandas` terminal
```

Comment:
I think I'd use `web/interactive_terminal` (not in `pandas/`). Things in `pandas/` are rendered as html, and probably better to nlt render this README and the rest of this directory.